src/test/java/io/pravega/connectors/flink/utils/SetupUtils.java

```diff
@@ -274,25 +302,29 @@ public void start() throws Exception {
 
             this.inProcPravegaCluster = InProcPravegaCluster.builder()
                     .isInProcZK(true)
+                    .secureZK(enableTls) //configure ZK for security
                     .zkUrl("localhost:" + zkPort)
                     .zkPort(zkPort)
                     .isInMemStorage(true)
                     .isInProcController(true)
                     .controllerCount(1)
                     .restServerPort(restPort)
+                    .enableRestServer(true)
```

Comment:
`enableRestServer` is being invoked twice here - line 312 and 319.

Reply:
Fixed